sdk/cognitivelanguage/ai-language-text/test/public/expectations.ts

```diff
@@ -8597,12 +8658,73 @@ export const expectation71: AnalyzeBatchResult[] = [
     completedOn,
     modelVersion,
   },
+  {
+    kind: "EntityLinking",
+    results: [
+      {
+        id: "0",
+        warnings: [],
+        entities: [],
+        detectedLanguage: { name: "English", iso6391Name: "en", confidenceScore: 0.98 },
+        isLanguageDefaulted: false,
+      },
+      {
+        id: "1",
+        error: {
+          code: "UnsupportedLanguageCode",
+          message:
+            "Invalid language code 'es'. Supported languages: en. For additional details see https://aka.ms/text-analytics/language-support?tabs=named-entity-recognition",
+        },
+      },
+      {
+        id: "2",
+        error: {
+          code: "UnsupportedLanguageCode",
+          message:
+            "Invalid language code 'ja'. Supported languages: en,es. For additional details see https://aka.ms/text-analytics/language-support?tabs=entity-linking",
+        },
+      },
+    ],
+    completedOn,
+    modelVersion,
+  },
+  {
+    kind: "Healthcare",
+    results: [
+      {
+        entities: [],
+        entityRelations: [],
+        id: "0",
+        warnings: [],
+        detectedLanguage: { "0": "e", "1": "n" },
```

Comment:
The generated layer was confused how to deserialize the string as an object and this is the result 😅

Reply:
I am tempted to write a swagger transform to make this a string and then we can parse it as an object or keep it as a string as needed in the convenience layer. I will look into this.

Reply:
Added in https://github.com/Azure/azure-sdk-for-js/pull/23720 😊